### ISSUE TITLE
pyre unable to find path

### ISSUE THREAD

**Issue by mrajancsr** [facebook/pyre-check#463]
# Pyre Bug

**Bug description**
Using virtual environment site-packages in search path...
ƛ Path does not exist: /Users/raj/Library/Python/3.9/lib/python/site-packages
ƛ Path does not exist: /Users/raj/Library/Python/3.9/lib/python/site-packages
ƛ Starting the server. Please wait... [120.1s]

This runs indefinitely

**Reproduction steps**
market-learn > marketlearn, py39

I usually do code market-learn to have my vscode start the project.  all the subdirectories under is what i want analyzed
location of market-learn directory
/Users/raj/Documents/QuantResearch/Home/market-learn

When i follow the installation steps, I cd into market-learn, then do pyre init.  After initializing watchman and using '.' for source directory, (which is market-learn), im left with the following configuration file: 

{
  "source_directories": [
    "."
  ],
  "taint_models_path": "/Users/raj/Documents/QuantResearch/Home/market-learn/py39/lib",
  "search_path": ["/Users/raj/Documents/QuantResearch/Home/market-learn/py39/lib/python3.9/site-packages/"]
}

I keep getting this warning: 
Path does not exist: /Users/raj/Library/Python/3.9/lib/python/site-packages

Im not too sure why pyre is checking a different path
**Expected behavior**
pyre picks up the correct path and reads all modules correctly

**Logs**
Please include any relevant logs here:

```
Output goes here
```

Please run your reproduction steps followed by `pyre rage > pyre_rage.log`, and upload the file here:

<FILE>
[pyre_rage.log](https://github.com/facebook/pyre-check/files/6987014/pyre_rage.log)

**Additional context**
I want pyre to recognize all my libraries

Is there any standard configuration you can share for macbook users using python 3.9?

**Comment by mrajancsr** [comment]
again not sure what happened. but it fixed it self.
